invenio_records_marc21/__init__.py

```diff
@@ -8,7 +8,107 @@
 # modify it under the terms of the MIT License; see LICENSE file for more
 # details.
 
-"""Marc21 datamodel."""
+"""Invenio-Records-Marc21 datamodel.
+
+This guide will show you how to get started with
+Invenio-Records-Marc21. It assumes that you already have knowledge of
```

Comment:
are the invenio modules also written like that? Are they all lowercase?

Reply:
In invenio modules using the upper case version. Invenio-modules get initialized with project_name and project_shortname. The value "project_shortname" is the lower case version of project_name. All docstrings and file headers are initialized with the project_name attribute. So, yes all modules in inveniosoftware should be written like this.